### PR TITLE
enh(ui): improve styling of unified view filters

### PR DESCRIPTION
**Pull request by kduret**
## Description

- add collapse
- increase width of global search
- put "filter" & "criterias" labels inlint

image

**Fixes** MON-5014

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,12 +69,13 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@centreon/ui": "centreon/centreon-ui",
+    "@centreon/ui": "github:centreon/centreon-ui",
```
Comment (bdauria) [inline package.json]: why this change?

File: www/front_src/src/Resources/Filter/index.tsx [hunk 2]
```diff
@@ -40,16 +53,38 @@ import {
...
+    transform: 'rotate(0deg)',
+    marginLeft: 'auto',
+    padding: theme.spacing(1),
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpenButton: {
+    transform: 'rotate(180deg)',
+  },
```
Comment (bdauria) [inline www/front_src/src/Resources/Filter/index.tsx]: I think we can live without the animation for now

File: www/front_src/src/Resources/index.test.tsx [hunk 3]
```diff
@@ -163,6 +172,20 @@ describe(Resources, () => {
...
   });
 
+  it('expands criterias filters', async () => {
+    const { queryByText } = render(<Resources />);
+
+    await wait(() => {
+      expect(queryByText(labelTypeOfResource)).not.toBeVisible();
+    });
+
+    expandCriteriasFilters();
```
Comment (bdauria) [inline www/front_src/src/Resources/index.test.tsx]: can we use `getByLabelText` instead?

File: package.json (hunk 1 above)
Comment (kduret) [reply package.json]: it's automatically added since I've upgraded npm

File: www/front_src/src/Resources/Filter/index.tsx (hunk 2 above)
Comment (kduret) [reply www/front_src/src/Resources/Filter/index.tsx]: Do you want I set transition duration to 0 ?

Comment (kduret) [reply www/front_src/src/Resources/Filter/index.tsx]: Done. I used ExpansionPanel component from material-ui
